### PR TITLE
foss4g 2023 post

### PR DESCRIPTION
**Pull request by jodygarnett**
Post thanks organizers, links to community posts, and provides a small update of code sprint activity.

### CODE REVIEW

File: _posts/2023-07-04-foss4g-2023.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+![](/img/posts/foss4g/prizren.jpeg) <br/>
+Prizren (Kosovo)
+
+## FOSS4G Code sprint
+
+We also had our first scheduled codesprint on removing the opengis-pending content from our codebase. The foss4g sprint was a fact-finding mission to establish the scope of work and establish a time frame and budget. 
+
+* The activity was successful in that we were able to change from *org.opengis* to *org.geotools.api* and write a script to update GeoWebCache and GeoServer projects.
+* GeoServer has dome difficulties as *gs-printing* and *gs-geofence-server* includes the work of external projects which will require assistance.
```
Comment (mprins) [inline _posts/2023-07-04-foss4g-2023.md:41]:
```suggestion
* GeoServer has some difficulties as *gs-printing* and *gs-geofence-server* include the work of external projects which will require assistance.
```

File: _posts/2023-07-04-foss4g-2023.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+* Many additional presentations featured GeoServer and showcased its capabilities.
+
+If videos are provided of the event we will update this post with links.
+
+![](/img/posts/foss4g/prizren.jpeg) <br/>
+Prizren (Kosovo)
+
+## FOSS4G Code sprint
+
+We also had our first scheduled codesprint on removing the opengis-pending content from our codebase. The foss4g sprint was a fact-finding mission to establish the scope of work and establish a time frame and budget.
```
Comment (bradh) [inline _posts/2023-07-04-foss4g-2023.md:38]: Not sure "opengis-pending" is well enough known to explain it. 

```suggestion
We also had our first scheduled codesprint on r[efactoring the OpenGIS](https://www.osgeo.org/opengis-harmonization/) content from our codebase. The foss4g sprint was a fact-finding mission to establish the scope of work and establish a time frame and budget. 
```

File: _posts/2023-07-04-foss4g-2023.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+Prizren (Kosovo)
+
+## FOSS4G Code sprint
+
+We also had our first scheduled codesprint on removing the opengis-pending content from our codebase. The foss4g sprint was a fact-finding mission to establish the scope of work and establish a time frame and budget. 
+
+* The activity was successful in that we were able to change from *org.opengis* to *org.geotools.api* and write a script to update GeoWebCache and GeoServer projects.
+* GeoServer has some difficulties as *gs-printing* and *gs-geofence-server* include the work of external projects which will require assistance.
+
+The work is scheduled for the [Bolsena Codesprint 2023](https://www.osgeo.org/events/bolsena-code-sprint-2023/) ahead of the GeoServer 2.23.0 release cycle.
```
Comment (bradh) [inline _posts/2023-07-04-foss4g-2023.md:43]: We are into 2.23.x already, I think. So 2.24?

```suggestion
The work is scheduled for the [Bolsena Codesprint 2023](https://www.osgeo.org/events/bolsena-code-sprint-2023/) ahead of the GeoServer 2.24.0 release cycle.
```

**Review by bradh — CHANGES_REQUESTED** [review]

File: _posts/2023-07-04-foss4g-2023.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+* [GeoServer Feature Frenzy 2023](https://docs.google.com/presentation/d/1vE8eCrOyewoH54g8CjuoiO3pxVLToEpuvpoZWmy0wTg/edit?usp=share_link)
+* [GeoServer used in fun and interesting ways](https://docs.google.com/presentation/d/1PP2qk7eH8TzAf1tvEWH7Geattd0YFh7ZEDx1_tlrRWY/edit?usp=share_link)
+* [GeoServer Orientation](https://docs.google.com/presentation/d/1sh9C4dIkDRnk3quCD1PRYoiJhjI9dqnAdOScJCgQWU8/edit?usp=share_link)
+* [Getting Started with GeoServer](https://docs.google.com/presentation/d/1FL6uMsepPWrrCMWjelIrlIp7bnlxB2q294-TN4lgI34/edit?usp=share_link) (Workshop)
+
+GeoServer community content:
+
+* Many additional presentations featured GeoServer and showcased its capabilities.
+
+If videos are provided of the event we will update this post with links.
```
Comment (ianturton) [inline _posts/2023-07-04-foss4g-2023.md:31]: when

**Review by ianturton — APPROVED** [review]

File: _posts/2023-07-04-foss4g-2023.md (hunk 4 above)
Comment (jodygarnett) [reply _posts/2023-07-04-foss4g-2023.md:31]: Please make the change directly

File: _posts/2023-07-04-foss4g-2023.md (hunk 2 above)
Comment (mprins) [reply _posts/2023-07-04-foss4g-2023.md:38]: `r[efactoring` ?

Comment (bradh) [reply _posts/2023-07-04-foss4g-2023.md:38]: I've fixed the link, but I'm not in love with the expression.

File: _posts/2023-07-04-foss4g-2023.md (hunk 4 above)
Comment (jodygarnett) [inline _posts/2023-07-04-foss4g-2023.md:31]:
```suggestion
When videos are provided of the event we will update this post with links.
```

**Pull request merged**